swift/ql/lib/codeql/swift/dataflow/ExternalFlow.qll

```diff
@@ -82,10 +82,12 @@ private module Frameworks {
   private import codeql.swift.frameworks.StandardLibrary.Data
   private import codeql.swift.frameworks.StandardLibrary.InputStream
   private import codeql.swift.frameworks.StandardLibrary.NSData
+  private import codeql.swift.frameworks.StandardLibrary.NsUrl
   private import codeql.swift.frameworks.StandardLibrary.String
   private import codeql.swift.frameworks.StandardLibrary.Url
   private import codeql.swift.frameworks.StandardLibrary.UrlSession
   private import codeql.swift.frameworks.StandardLibrary.WebView
+  private import codeql.swift.security.PathInjection
```

Comment:
It feels like this import belongs be somewhere else?

Reply:
This is needed for the MaD sinks present in `PathInjection.qll` to be added to the global scope, so that flow paths for all queries are only computed once. Otherwise, running the path injection query would require (AFAIK) some recomputation, impacting performance.